students/mariusz_michalczyk/lesson_05_lists/numbers_filter.py

```diff
@@ -0,0 +1,8 @@
+strs = ['2', '0', '8', '3']
+
+
+def filter_nrs(strs, range_to_filter):
+    return [i for i in strs[range_to_filter - 1:len(strs)]]
+
+
+print(filter_nrs(strs, 3))
```

Comment:
I think Janusz (I guess that was from him, as he answered in the channel) meant something different. He meant to filter _out_ range(3) which is 0, 1 and 2, and hence 0 and 2 are out. There were no specs here and not even a function call in an example, so I don't care... but you would probably get something around "TypeError: can't substract string from a generator".